package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "angular-translate-loader",
+  "version": "1.1.1",
```

Comment:
@Ardeshir81 `version` will be automatically added by [`semantic-release `](https://github.com/semantic-release/semantic-release).

Could you please delete it?